drivers/hmis/app/controllers/hmis/app_settings_controller.rb

```diff
@@ -10,8 +10,24 @@ class Hmis::AppSettingsController < Hmis::BaseController
 
   def show
     okta_enabled = ENV['HMIS_OKTA_CLIENT_ID'].present? && ENV['OKTA_DOMAIN'].present?
+
+    logo = ENV['LOGO']
+    logo_path = if SerializedAsset.exists?(logo)
+      SerializedAsset.get_src(logo)
+    else
+      "theme/logo/#{logo}"
+    end
+
+    hostname = ENV['FQDN']
+
     render json: {
       oktaPath: okta_enabled ? '/hmis/users/auth/okta' : nil,
+      logoPath: logo_path.present? ? ActionController::Base.helpers.asset_path(logo_path) : nil,
+      warehouseUrl: "https://#{hostname}",
+      warehouseName: _('Boston DND Warehouse'),
+      resetPasswordUrl: "https://#{hostname}/users/password/new",
+      unlockAccountUrl: "https://#{hostname}/users/unlock/new",
+      casUrl: nil, # TODO: get CAS url from env
```

Comment:
Needed if we want to keep the link to CAS on the top bar (currently only enabled in QA). @eanders should I just add an env var for it?

Reply:
not sure exactly what you need, but you should be able to determine if we need to show the CAS link if `ENV['DATABASE_CAS_DB']` isn't empty.  CAS url should be available via `Config.get(: cas_url)` Would those be sufficient?

Reply:
Yes, thanks